app/views/agent/NextUpdates.scala.html

```diff
@@ -19,7 +19,7 @@
 
 @this(mainTemplate: MainTemplate, nextUpdatesHelper: NextUpdatesHelper)
```

Comment:
Question: Does the agent page look okay still? It hasn't been migrated yet to new layout but already uses the "migrated" `NextUpdatesHelper` with new classes there. Is everything okay on the agent page?

Reply:
Checked agent NextUpdates, added in a extra "visuallyhidden" class to the "Open all" to hide the "sections" text that was appearing. Looks visually fine